staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2862,6 +2862,31 @@ type PodDNSConfigOption struct {
 	Value *string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// EphemeralContainer describes a container to attach to a running pod for troubleshooting.
+// This field is alpha-level and is only honored by servers that enable the DebugContainers feature.
```

Comment:
"This field" should probably be rephrased given that this is commenting on a type, not a field.